integration-tests/smoke/ccip_test.go

```diff
@@ -112,21 +113,16 @@ func TestInitialDeployOnLocal(t *testing.T) {
 
 func TestTokenTransfer(t *testing.T) {
 	t.Parallel()
-	lggr := logger.TestLogger(t)
-	ctx := ccdeploy.Context(t)
-	tenv, _, _ := testsetups.NewLocalDevEnvironmentWithDefaultPrice(t, lggr)
-
+	tenv, state := setupEnvironment(t)
```

Comment:
Can you rebase? This has been extracted out in another PR and you can get rid of `setupEnvironment`